tests/jobs/test_job.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+
+from snuba.manual_jobs import FINISHED, NOT_STARTED, JobSpec, get_job_status
+from snuba.manual_jobs.toy_job import ToyJob
+
+JOB_ID = "abc1234"
+test_job_spec = JobSpec(job_id=JOB_ID, job_type="ToyJob")
+
+# @pytest.fixture(autouse=True)
```

Comment:
it turns out the state of the redis client does not persist outside, so after inserting ['abc1234', 'not started'] during init of ToyJob, redis_client.hkeys would show nothing inserted in `test_job_status_changes_to_finished`